test/common/json-model-parser.spec.js

```diff
@@ -0,0 +1,189 @@
+describe('json-model-parser', function () {
+    'use strict';
+    let requireJs = require('requirejs');
```

Comment:
`const requirejs = require('webgme').requirejs;` would be better so you don't have to call `config`

Reply:
Changed it to use webgme's requirejs.
```javascript
const requireJs = require('webgme').requirejs;
const ModelParser = requireJs('../../../../../src/plugins/ImportKeras/utils/json-model-parser');
```